docs/userguide/configuration.rst

```diff
@@ -511,6 +512,41 @@ Default: No time limit.
 Task hard time limit in seconds. The worker processing the task will
 be killed and replaced with a new one when this is exceeded.
 
+.. setting:: task_allow_error_cb_on_chord_header
+
+.. _task_allow_error_cb_on_chord_header:
+
+``task_allow_error_cb_on_chord_header``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.3
+
+Default: Disabled.
+
+Enabling this flag will allow linking an error callback to a chord header,
+which by default will not link when using :code:`link_error()`, and preventing
+from the chord body to execute if the header fails.
+
+Consider the following canvas:
+
+.. code-block:: python
+
+    header = group([t1, t2])
+    body = t3
+    c = chord(header, body)
+    c.link_error(error_callback_sig)
+
+If :code:`t1` or :code:`t2` will fail, by default, the chord body would still execute, and :code:`error_callback_sig` will **not** be called.
```

Comment:
Have we verified this?

Reply:
Now I did. This description is wrong. Will be fixed.

Reply:
Fix pushed